src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -42,6 +48,7 @@ public partial class TextEditor : View
         public TextEditor() : this(Interop.TextEditor.New(), true)
         {
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            initializeTextEditor();
```

Comment:
This is duplicate of line:86

Reply:
The whole initializeText****() functions are not necessary if we decide to do not use null-able property.
I will remove it.